app/assets/js/app/modules/analytics.js

```diff
@@ -1,16 +1,13 @@
 import domready from './domready'
 
 export let trackEvent = (event, data) => {
-  console.log(`[Analytics disabled] Event: ${event}`)
-  console.log(data)
+  // No-op
```

Comment:
This is really useful for debugging analytics events. I think just adding some `eslint-ignore` comments is better than removing it altogether.